generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management-update.component.html.ejs

```diff
@@ -211,9 +211,10 @@ _%>
             </div>
 
             <div>
-                <button pButton type="button" id="cancel-save" icon="pi pi-ban" class="ui-button-secondary" (click)="previousState()" [label]="'entity.action.cancel' | translate"></button>
-                <button pButton type="submit" id="save-entity" icon="pi pi-save" [disabled]="editForm.invalid || isSaving" [label]="'entity.action.save' | translate"></button>
+                <br />
+                <button pButton type="button" id="cancel-save" icon="pi pi-ban" class="ui-button-secondary" (click)="previousState()" [label]="'entity.action.cancel' | translate" class="btn btn-secondary"></button>
```

Comment:
The class attribute is repeated.
Consider putting all classes in one `class` attribute